app/models/account.rb

```diff
@@ -33,6 +33,15 @@ def transactions
   end
 
   def accumulative_balance_data
-    transactions.group_by_day(:created_at).sum(:amount)
+    transactions_grouped_by_day = transactions.group_by_day(:created_at).sum(:amount)
```

Comment:
It still doesn't count in the negative transactions (not all transactions are top-up accounts!).
Can we use SQL for this and delete this gem for group_by_day?